capstone/src/main/resources/static/user.js

```diff
@@ -1,4 +1,7 @@
-/* helper function to highlight active tab for general nav */
+/**
+  * helper function to highlight active tab for general nav
```

Comment:
You can skip "helper function", and instead go with 3rd person -- JSdoc has fewer hard rules than Javadoc, but it's going to look familiar to other developers to see "Highlights active tab in general navbar".

As an example, here is Google JSdoc style guide: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments